experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1257,61 +1257,121 @@ def test_no_treatment_branches_for_rollout(self):
             [NimbusConstants.ERROR_SINGLE_BRANCH_FOR_ROLLOUT],
         )
 
-    def test_rollout_valid_version_support(self):
-        desktop = NimbusExperiment.Application.DESKTOP
+    @parameterized.expand(
+        [
+            (
+                NimbusExperiment.Application.DESKTOP,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MAC_ONLY,
+            ),
+            (
+                NimbusExperiment.Application.FENIX,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MOBILE_NEW_USERS,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_ANDROID,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MOBILE_NEW_USERS,
+            ),
+            (
+                NimbusExperiment.Application.IOS,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MOBILE_NEW_USERS,
+            ),
+            (
+                NimbusExperiment.Application.FOCUS_IOS,
+                NimbusExperiment.Version.FIREFOX_105,
+                NimbusExperiment.TargetingConfig.MOBILE_NEW_USERS,
+            ),
+        ]
+    )
+    def test_rollout_valid_version_support(
+        self, application, firefox_version, targeting_config
+    ):
         experiment = NimbusExperimentFactory.create_with_lifecycle(
             NimbusExperimentFactory.Lifecycles.CREATED,
-            application=desktop,
+            application=application,
             channel=NimbusExperiment.Channel.NO_CHANNEL,
-            firefox_min_version=NimbusExperiment.Version.FIREFOX_108,
-            feature_configs=[NimbusFeatureConfigFactory(application=desktop)],
+            firefox_min_version=firefox_version,
+            feature_configs=[NimbusFeatureConfigFactory(application=application)],
             is_sticky=False,
```

Comment:
It is false here

Reply:
This should be True,
on checking, `sticky_required`  is `True` for MOBILE_NEW_USER and `False` for MAC_ONLY, should I update this to `True`

Reply:
if it's different then consider passing this as a parameter too and in the parameter you can pass true false value or the other option is to find a targeting config which has `sticky_required` true for the desktop

Reply:
I've updated the targeting configs for desktop to one which has `sticky_required` true, with this now the tests are okay.